logic/transaction.js

```diff
@@ -144,8 +145,16 @@ Transaction.prototype.getBytes = function (trs, skipSignature, skipSecondSignatu
 			}
 		}
 
+
 		bb.writeLong(trs.amount);
 
+		if (trs.data) {
+			var dataBuffer = new Buffer(trs.data, 'utf8');
+			for (i = 0; i < 64; i++) {
+				bb.writeByte(dataBuffer[i] || 0);
```

Comment:
@karmacoma what do you think about `||` expression in this statement? Should I change this loop to something like: 
`for (i = 0; i < dataBuffer.length; i++) {
       bb.writeByte(dataBuffer[i]);
}`